pom.xml

```diff
@@ -81,24 +83,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
-        <version>3.1.0</version>
+        <version>3.1.1</version>
         <configuration>
-          <configLocation>checkstyle.xml</configLocation>
+          <configLocation>google_checks.xml</configLocation>
           <encoding>UTF-8</encoding>
           <consoleOutput>true</consoleOutput>
-          <!-- Enable this once all warnings are resolved
           <violationSeverity>warning</violationSeverity>
           <failsOnError>true</failsOnError>
-          -->
           <linkXRef>false</linkXRef>
+          <suppressionsLocation>checkstyle-suppressions.xml</suppressionsLocation>
+          <suppressionsFileExpression>checkstyle.suppressions.file</suppressionsFileExpression>
+          <includeTestSourceDirectory>true</includeTestSourceDirectory>
         </configuration>
         <executions>
           <execution>
             <id>validate</id>
-            <phase>validate</phase>
+            <phase>package</phase>
```

Comment:
Just curious, what does this change do?

Reply:
The `validate` phase goes before the `package` stage. In particular, the `validation` phase is run before the tests run. In means that `checkstyle` runs before tests, and if there are any style issues, then the build stops before running tests. Now imagine that you develop a new feature and want to run tests quickly. If you introduced some style issues, you'll have to fix them in order to run your tests. That may be annoying. If checkstyle runs in the `package` phase, then the tests run before. That's why I updated the phase.

Reply:
Great, thanks for the explanation!